src/vue-pdf-embed.vue

```diff
@@ -63,6 +63,14 @@ export default {
      * @values Number, String
      */
     width: [Number, String],
+    /**
+     * The page rotation in degrees, only multiple of 90 are valid. EG: 90, 180, 270, 360, ...
```

Comment:
And the following comment here:

```suggestion
     * Desired page rotation angle.
```